packages/driver-interface/src/services/messaging.ts

```diff
@@ -34,23 +36,51 @@ export const onDriverLoginSuccessful = (
     )
   )
 
+export const sendSummary = (
+  telegramId: number,
+  instructions: Instruction[][]
```

Comment:
Same as before, often named instructionGroups, to not confuse with Instruction[]